src/components/form/fields/RadioGroupField.tsx

```diff
@@ -44,12 +44,11 @@ export function RadioGroupField({
           <FormLabel>{label}</FormLabel>
           <FormControl>
             <RadioGroup
-              // onValueChange={field.onChange}
               onValueChange={(value: string) => {
                 field.onChange(value);
                 onChange?.(value);
               }}
-              defaultValue={field.value}
+              value={field.value}
```

Comment:
@gabrielseco I had to make this change so that RadioGroup is updated when the initial values are updated (relate to my comment in you [PR](https://github.com/remoteoss/remote-flows/pull/73#discussion_r2043969448) 🙂  )